web/template/partial/course/manage/create-lecture-form.gohtml

```diff
@@ -1,138 +1,117 @@
+            <span class="text-sm text-5">Video File (mp4, if possible h264)</span>
+            <input type="file" accept="video/mp4" class="btn"
+                   x-on:change="formData.file = Object.values($event.target.files)">
+        </label>
+        <label x-show="!formData.premiere && !formData.vodup && !loading">
+            <span class="text-sm text-5">Lecture Hall</span>
+            <select class="tl-select" id="lectureHallBulkSelect" x-model="formData.lectureHallId">
+                <option class="text-gray-300" value="0">Self streaming
+                </option>
+                {{range .LectureHalls}}
+                    <option value="{{.Model.ID}}"
+                            class="text-gray-300">
+                        {{.Name}}
+                    </option>
+                {{end}}
+            </select>
+        </label>
+        <div class="flex" x-show="formData.recurring && !loading">
+            <div class="w-1/4 pl-2">
+                <span class="font-semibold mb-6">Interval</span><br>
+                <label>
+                    <input type="radio" value="daily" x-model="formData.recurringInterval" class="w-auto"
+                           @change="regenerateRecurringDates">
+                    <span class="text-3">Repeat Daily</span>
+                </label><br>
                 <label>
-                    <input type="checkbox" x-model="formData.vodup" @change="(v) => v && (formData.recurring = false)"
-                           class="w-auto">
-                    <span class="text-3">VoD Upload</span>
+                    <input type="radio" value="weekly" x-model="formData.recurringInterval" class="w-auto"
+                           @change="regenerateRecurringDates">
+                    <span class="text-3">Repeat Weekly</span>
                 </label><br>
                 <label>
-                    <input type="checkbox" :disabled="formData.premiere || formData.vodup" x-model="formData.recurring"
-                           class="w-auto">
-                    <span class="text-3">Recurring Event</span>
+                    <input type="radio" value="monthly" x-model="formData.recurringInterval" class="w-auto"
+                           @change="regenerateRecurringDates">
+                    <span class="text-3">Repeat Monthly</span>
                 </label><br>
-                <label x-show="{{$course.ChatEnabled}}">
-                    <input type="checkbox" x-model="formData.isChatEnabled" x-init="formData.isChatEnabled = {{$course.ChatEnabled}}"
-                           class="w-auto">
-                    <span class="text-3">Enable Live Chat</span>
+            </div>
+            <div class="w-1/4 pl-2">
+                <label for="events-count" class="w-full">
+                    <span class="font-semibold p-2">Count</span>
+                    <input name="events-count" type="number" placeholder="14" id="end" min="1"
```

Comment:
In light-mode, the count-input-field doesn't look like an input field. I am not sure if `tl-input` fits but i guess it's worth a try.

<img width="187" alt="Screenshot 2022-10-17 at 06 52 13" src="https://user-images.githubusercontent.com/29633518/196092950-e39a8295-5513-43e3-be0c-d175bddc8d86.png">